src/driver/communication_interface.cc

```diff
@@ -387,6 +387,7 @@ bool CommunicationInterface::ModeIsValid(
     case franka::RobotMode::kAutomaticErrorRecovery:
       return true;
   }
+  log()->error("ModeIsValid: got invalid mode: {}", current_mode);
```

Comment:
need to convert `current_mode` to string with `RobotModeToString`

Reply:
No, that won't work, because it is not a valid mode. It does not exist in the enum of known modes.

Reply:
I want the raw integer value

Reply:
Yep you are right. Do you need a cast here?